src/ufonormalizer.py

```diff
@@ -1293,7 +1294,7 @@ def _plistDate(self, data):
         self.simpleElement("date", value=data)
 
     def _plistData(self, data):
-        data = data.asBase64(maxlinelength=xmlTextMaxLineLength)
+        data = base64.b64encode(data)
```

Comment:
here you're losing the line wrapping when length exceeds `xmlTestMaxLineLength`.
You should implement something like this and use it here:
https://github.com/python/cpython/blob/3689c25a1010c2acdb05f1b1b0229f4766b4440a/Lib/plistlib.py#L112-L119

Reply:
and  maybe also add a tests that exercises this line wrapping, because current tests didn't catch that

Reply:
> Anyone have concerns about that?

I don't have any concerns.

Reply:
None either, remove away!